src/types.ts

```diff
@@ -1,11 +1,37 @@
-export { Request as ExpressRequest } from 'express';
+type Timestamp = number;
 
```

Comment:
The types that are more expected to be imported by consumers of this library are here.

Other types like function arguments and return types were kept in the same file as the method.